public/museu/evolution-of-tabnews.html

```diff
@@ -0,0 +1,349 @@
+        </blockquote>
+        <h2 id="3-25-12-2020">3. 25/12/2020</h2>
+        <p>Estamos exatamente na data de criação do repositório, e agora você sabe que o repositório do TabNews foi criado no natal. 🎅🏻</p>
+        <h2 id="4-30-12-2020">4. 30/12/2020</h2>
+        <p>A primeira <abbr title="O Issue é uma ferramenta interessante do GitHub para deixar um feedback ou auxiliar na construção de algum projeto desenvolvido por outras pessoas. Trata-se da ideia de deixar comentários em projetos.">issue</abbr> foi criada no repositório. Foi uma issue de inception, fala sobre o porquê o projeto foi criado, de onde o nome veio e outras coisas. Se quiser, você pode ler ela <a href="https://github.com/filipedeschamps/tabnews.com.br/wiki/004.-Primeira-issue-e-a-descoberta-de-algo-importante">clicando aqui.</a></p>
+        <h2 id="5-07-05-2021">5. 07/05/2021</h2>
+        <p>A primeira idéia de layout</p>
+        <div align="center"><img src="https://user-images.githubusercontent.com/4248081/117865517-e6891680-b24a-11eb-8ba8-fddd4f3f26de.png"></div>
+
+        <h2 id="6-11-05-2021">6. 11/05/2021</h2>
+        <p>O primeiro <abbr title="PR são as iniciais de Pull Request. Geralmente em projetos onde não temos permissão para alterar a branch principal (geralmente a master), ainda assim podemos criar alguma modificação local e participar do seu desenvolvimento.">PR</abbr> no repositório. </p>
+        <p>Foi um PR com o seguinte título: <code>layout: trying a more &quot;Twitter&quot; style</code></p>
+        <div align="center"><img src="https://i.imgur.com/dXACkkr.png"></div>
+
+        <h2 id="7-entre-11-05-e-13-05-2021">7. Entre 11/05 e 18/05/2021</h2>
+        <p>Idéia de layout número 4, desenvolvido pelo Filipe:</p>
+        <div align="center"><img src="https://user-images.githubusercontent.com/4248081/118160451-2c70e680-b3d3-11eb-9f6e-96cd1b579811.png"></div>
+
+        <p>Até que no dia 13, o <a href="https://github.com/filipedeschamps/tabnews.com.br/wiki/014.-Guga-e-Renata-tiveram-excelentes-ideias">Guga e a Renata tiveram excelentes idéias</a>. (se não quiser acessar o link, vou tentar resumir aqui em baixo):
+        O Filipe estava desenvolvendo esse design, só que o Guga e ele acharam que tinha muita informação, ficou muito confuso. Depois, ao mostrar o HackerNews pra Renata, ela disse que o legal era ler as notícias de forma <strong>linear</strong>, que não havia quebra de linha e isso facilitava o scan.</p>
+        <h3 id="13-05-a-18-05-2021">13/05 a 18/05/2021</h3>
+        <p>Todas as (eu acho, se não pelo menos as principais) idéias de layout do Filipe. E realmente, a última é até que bastante parecida com a atual. </p>
```

Comment:
Aqui acho que a leitura fica mais fluida trocando `se não pelo menos` por `ou ao menos` ou `ou pelo menos` mesmo, trocando apenas o `se não` por `ou`.

```suggestion
        <p>Todas as (eu acho, ou ao menos as principais) ideias de layout do Filipe. E realmente, a última é até que bastante parecida com a atual.</p>
```